CHANGES.md

```diff
@@ -7,7 +7,7 @@ Airbase 129
   - modernizer 2.4.0 (from 2.3.0)
   - joda-time 2.10.14 (from 2.10.13)
   - slf4j 1.7.36 (from 1.7.32)
-  - checkstyle 10.3.2 (from 9.2)
+  - checkstyle 10.3.3 (from 9.2) and maven-checkstyle-plugin 3.2.0 (from 3.1.2)
```

Comment:
The "from" version is wrong, should be 10.3.2

Reply:
It's the same case as with joda-time, it was recently updated and was not released yet.